.github/workflows/deploy.yml

```
@@ -209,8 +171,10 @@ jobs:
 # JOB 2: DEPLOY
 # ═══════════════════════════════════════════════════════════════════════════════
 
+  # --- ETAPA 2: DEPLOY VIA SSH ---
   deploy:
-    name: 🚀 Deploy
+    name: 🚚 Deploy via SSH para Hostinger
+    runs-on: ubuntu-latest
     needs: build
     runs-on: ubuntu-24.04
```

actionlint (1.7.9)

key "runs-on" is duplicated in "deploy" job. previously defined at line:177,col:5

(syntax-check)